src/home/components/home-banner/index.scss

```diff
@@ -0,0 +1,37 @@
+.home-banner {
+    height: var(--home-page-banner-height, 27.5rem);
```

Comment:
Let's start all the CSS variables we add with `catalog` so that it is clear that these variables belong to this MFE Catalog

```suggestion
    height: var(--catalog-home-page-banner-height, 27.5rem);
```